modules/representation_management/app/swagger/v0/swagger.json

```diff
@@ -331,6 +327,22 @@
                       }
                     }
                   },
+                  "representative": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "example": "123456789"
+                      },
+                      "organization_id": {
+                        "type": "string",
+                        "example": "Veterans Organization"
```

Comment:
`Veterans Organization` is a misleading example. It should be an example uuid or a poa code

Reply:
Fixed by regenerating the docs after I changed the code.